src/plugins/plugin.filler.js

```diff
@@ -354,14 +357,15 @@ function _fill(ctx, cfg) {
 
 		ctx.beginPath();
 
-		let loop = !!line.pathSegment(ctx, src);
-		if (loop) {
+		const lineLoop = !!line.pathSegment(ctx, src);
+		if (lineLoop) {
 			ctx.closePath();
 		} else {
 			interpolatedLineTo(ctx, target, end, property);
 		}
 
-		loop &= target.pathSegment(ctx, tgt, {move: loop, reverse: true});
+		const targetLoop = !!target.pathSegment(ctx, tgt, {move: lineLoop, reverse: true});
```

Comment:
Just FYI:
Could have actually done it like this (just reverse your initial try, so the short-circuiting does not have any effect):
```js
loop = target.pathSegment(ctx, tgt, {move: loop, reverse: true}) && loop;
```